tests/realtikvtest/txntest/stale_read_test.go

```diff
@@ -140,16 +140,14 @@ func TestSelectAsOf(t *testing.T) {
 	store := realtikvtest.CreateMockStoreAndSetup(t)
 	tk := testkit.NewTestKit(t, store)
 	tk.MustExec("use test")
-	if !*realtikvtest.WithRealTiKV {
-		// For mocktikv, safe point is not initialized, we manually insert it for snapshot to use.
-		safePointName := "tikv_gc_safe_point"
-		safePointValue := "20160102-15:04:05 -0700"
-		safePointComment := "All versions after safe point can be accessed. (DO NOT EDIT)"
-		updateSafePoint := fmt.Sprintf(`INSERT INTO mysql.tidb VALUES ('%[1]s', '%[2]s', '%[3]s')
+	// For mocktikv, safe point is not initialized, we manually insert it for snapshot to use.
```

Comment:
Is this still mocktikv-only? Or is it needed for real tikv?

Reply:
It's needed for real TiKV as well. The comments are deleted

Reply:
I'm afraid it doesn't look like a good idea to do this on real tikv. But let's accept it for now as it would be too complex to do it elegantly.